keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
+        """Count the total number of floating point operations for a _forward_ calculation of this layer.
```

Comment:
Some of your lines are too long. We don't strictly enforce line length, but please do your best to follow PEP8.